diff --git a/libgsmhr/fetch_sources.py b/libgsmhr/fetch_sources.py
index 6fb1b8e..545bd3c 100755
--- a/libgsmhr/fetch_sources.py
+++ b/libgsmhr/fetch_sources.py
@@ -10,8 +10,7 @@ try:
 except:
 	import StringIO
 
-
-SRC = "http://www.3gpp.org/ftp/Specs/archive/06_series/06.06/0606-421.zip"
+SRC = "../0606_421.zip"
 HDR = {
 	"User-Agent": "Mozilla/5.0 (X11; ArchLinux; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0",
 	"Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
@@ -49,9 +48,8 @@ def main(*args):
 	os.chdir(tgt)
 
 	# Get the original data
-	req = urllib2.Request(SRC, headers = HDR)
-	u = urllib2.urlopen(req)
-	d = u.read()
+	f = open(SRC)
+	d = f.read()
 
 	# Get DISK.zip
 	d = get_subfile_data(d, 'DISK.zip')
