diff -ru app.orig/helpers/application_helper.rb app/helpers/application_helper.rb
--- app.orig/helpers/application_helper.rb      2018-01-08 19:37:37.000000000 +0000
+++ app/helpers/application_helper.rb   2018-05-14 08:15:11.558390732 +0000
@@ -637,7 +637,7 @@
     attachments = options[:attachments] || []
     attachments += obj.attachments if obj.respond_to?(:attachments)
     if attachments.present?
-      text.gsub!(/src="([^\/"]+\.(bmp|gif|jpg|jpe|jpeg|png))"(\s+alt="([^"]*)")?/i) do |m|
+      text.gsub!(/src="([^\/"]+\.(bmp|gif|jpg|jpe|jpeg|png|svg))"(\s+alt="([^"]*)")?/i) do |m|
         filename, ext, alt, alttext = $1.downcase, $2, $3, $4
         # search for the picture in attachments
         if found = Attachment.latest_attach(attachments, CGI.unescape(filename))
diff -ru app.orig/models/attachment.rb app/models/attachment.rb
--- app.orig/models/attachment.rb       2018-01-08 19:37:37.000000000 +0000
+++ app/models/attachment.rb    2018-05-14 08:14:49.542526978 +0000
@@ -193,7 +193,7 @@
   end
 
   def image?
-    !!(self.filename =~ /\.(bmp|gif|jpg|jpe|jpeg|png)$/i)
+    !!(self.filename =~ /\.(bmp|gif|jpg|jpe|jpeg|png|svg)$/i)
   end
 
   def thumbnailable?
