--- libdbi-drivers/configure.ac.orig	2016-02-04 10:12:56.049468401 +0100
+++ libdbi-drivers/configure.ac	2016-02-04 10:13:10.336666283 +0100
@@ -3,7 +3,7 @@
 AC_CONFIG_SRCDIR([drivers/sqlite/dbd_sqlite.c])
 AM_MAINTAINER_MODE
 
-AM_INIT_AUTOMAKE([foreign dist-zip subdir-objects serial-tests])
+AM_INIT_AUTOMAKE([foreign dist-zip subdir-objects])
 AM_CONFIG_HEADER(config.h)
 AC_LIBTOOL_WIN32_DLL
 AC_CONFIG_MACRO_DIR([m4])
